driver-core/src/main/java/com/datastax/driver/core/DataTypeClassNameParser.java

```diff
@@ -113,7 +113,7 @@ static DataType parseOne(String className, ProtocolVersion protocolVersion, Code
             List<UserType.Field> fields = new ArrayList<UserType.Field>(rawFields.size());
             for (Map.Entry<String, String> entry : rawFields.entrySet())
                 fields.add(new UserType.Field(entry.getKey(), parseOne(entry.getValue(), protocolVersion, codecRegistry)));
-            return new UserType(keyspace, typeName, fields, protocolVersion, codecRegistry);
+            return new UserType(keyspace, typeName, false, fields, protocolVersion, codecRegistry);
```

Comment:
With regards to [this comment](https://github.com/datastax/java-driver/pull/824#discussion_r104987494) I think we should have this pass in true since this is only used for < C* 3.0, where udts are always frozen.  I'll push that up but feel free to back it out if you disagree.